# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: build/sparse-profiles/github-sync
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/build/sparse-profiles/github-sync b/build/sparse-profiles/github-sync
--- a/build/sparse-profiles/github-sync
+++ b/build/sparse-profiles/github-sync
@@ -1,8 +1,7 @@
 %include build/sparse-profiles/mach
 %include build/sparse-profiles/taskgraph
 
 [include]
-path:gfx/wgpu/
 path:gfx/wr/
 path:taskcluster/scripts/misc/
 path:tools/github-sync/